package.json

```diff
@@ -3,10 +3,26 @@
   "version": "0.0.1",
   "license": "UNLICENSED",
   "scripts": {
-    "android": "react-native run-android",
-    "android-release": "yarn android --mode release",
-    "android:bundle:release": "npx react-native build-android --mode=release",
-    "ios": "react-native run-ios",
+    "android": "yarn android-dev",
+    "android-dev": "ruby scripts/launch-android --env=dev",
+    "android-prod": "ruby scripts/launch-android --env=prod",
+    "android-dev-release": "ruby scripts/launch-android --env=dev --release",
+    "android-prod-release": "ruby scripts/launch-android --env=prod --release",
+    "android-clean": "yarn android-dev-clean",
+    "android-dev-clean": "ruby scripts/launch-android --env=dev --clean",
+    "android-prod-clean": "ruby scripts/launch-android --env=prod --clean",
+    "android-dev-release-clean": "ruby scripts/launch-android --env=dev --release --clean",
+    "android-prod-release-clean": "ruby scripts/launch-android --env=prod --release --clean",
+    "android:bundle:dev-release": "cd android && ./gradlew bundleDevRelease",
+    "android:bundle:prod-release": "cd android && ./gradlew bundleProdRelease",
+    "ios": "yarn ios-dev",
+    "ios-dev": "yarn ios-dev-sim",
+    "ios-prod": "yarn ios-prod-sim",
+    "ios-device": "yarn ios-dev-device",
+    "ios-dev-sim": "react-native run-ios --scheme=freighter-mobile-dev --simulator=\"$(./scripts/get-ios-simulator)\"",
+    "ios-prod-sim": "react-native run-ios --scheme=freighter-mobile --simulator=\"$(./scripts/get-ios-simulator)\"",
+    "ios-dev-device": "react-native run-ios --scheme=freighter-mobile-dev --device",
+    "ios-prod-device": "react-native run-ios --scheme=freighter-mobile --device",
```

Comment:
Those are all the new script commands to help with running both apps (`Prod` or `Dev`) in both modes (`Debug` or `Release`). It also make it easy to run on iOS Simulator or iPhone Device.